packages/ncids-js/src/index.ts

```diff
@@ -7,20 +7,23 @@ export { NCISubscribe } from './components/nci-subscribe/nci-subscribe.component
 import { NCIExtendedHeaderWithMegaMenu } from './components/nci-header/extended-with-mega-menu/index';
 export { NCIExtendedHeaderWithMegaMenu };
 
-import type { MegaMenuAdaptor } from './components/nci-header/index';
-export type { MegaMenuAdaptor };
+import type { MegaMenuAdapter as MegaMenuAdaptor } from './components/nci-header/mega-menu';
 
-import type { MobileMenuAdaptor } from './components/nci-header/index';
-export type { MobileMenuAdaptor };
-
-import type { MobileMenuData } from './components/nci-header/index';
-export type { MobileMenuData };
+import type {
```

Comment:
should we also note here that `Adapter` is the correct implementation and that this will go away in v3.0?

Reply:
Actually this entire index.ts should be deprecated I think. However, good point on needing a way to note that.

Reply:
So I marked these as deprecated, but it does not seem to do anything in the built types.